### PR TITLE
updating JSON example to match XML example - CF-705

### PR DESCRIPTION
**Pull request by stanzikratel**

### CODE REVIEW

**Comment by usnavi** [comment]
I think the changes submitted here belong in AtomFeedElementExampleexternal.json?  It has the tenant id in the links, which is only for external.  I believe AtomFeedElementExample.json & AtomFeedElementExampleexternal.json both need to be updated.

**Comment by shintasmith** [comment]
the indenting seems to be messed up too... I'm not sure if that will show up in PDF or not.

**Comment by stanzikratel** [comment]
Updated external example to remove all occurrences of tenant id and reformatted JSON to ensure it displays correctly in build

**Comment by usnavi** [comment]
the tid:123456 category should still exist for all entries in the non-external example.

**Comment by stanzikratel** [comment]
I reinserted the TID value for internal example

**Comment by usnavi** [comment]
lgtm

File: src/docbkx/samples/AtomFeedElementExample.json [hunk 1]
```diff
@@ -1,25 +1,166 @@
...
+                 {
+                   "term": "dc:DFW1" 
+                 }
+              ],        
+           "atom:content": 
+             {         
+               "type": "application/xml",         
+               "event": 
+                {           
+                  "xmlns:widget": "http://docs.rackspace.com/usage/widget",
```
Comment (shintasmith) [inline src/docbkx/samples/AtomFeedElementExample.json]: @usnavi, that ```xmlns:widget``` looks strange.

**Comment by usnavi** [comment]
@shintasmith noticed that the XML & JSON were all jacked up.  Good catch.  Closing and replacing it with another PR which fixes XML & JSON

**Pull request closed without merging**